app-i18n/uim/uim-1.8.6-r2.ebuild

```diff
@@ -0,0 +1,228 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI="5"
+inherit autotools eutils multilib elisp-common gnome2-utils
+
+DESCRIPTION="Simple, secure and flexible input method library"
+HOMEPAGE="https://github.com/uim/uim/"
+SRC_URI="https://uim.googlecode.com/files/${P}.tar.bz2"
+
+LICENSE="BSD GPL-2 LGPL-2.1"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~hppa ~ppc ~ppc64 ~x86"
+IUSE="+anthy canna curl eb emacs expat libffi gtk gtk3 kde libedit libnotify libressl m17n-lib ncurses nls qt4 skk sqlite ssl static-libs test unicode X xft l10n_ja l10n_ko l10n_zh-CN l10n_zh-TW"
+
+RESTRICT="test"
+
+REQUIRED_USE="gtk? ( X ) qt4? ( X )"
+
+RDEPEND="X? (
+		x11-libs/libX11
+		x11-libs/libXft
+		x11-libs/libXt
+		x11-libs/libICE
+		x11-libs/libSM
+		x11-libs/libXext
+		x11-libs/libXrender
+	)
+	anthy? (
+		unicode? ( >=app-i18n/anthy-8622 )
+		!unicode? ( app-i18n/anthy )
+	)
+	canna? ( app-i18n/canna )
+	curl? ( >=net-misc/curl-7.16.4 )
+	eb? ( dev-libs/eb )
+	emacs? ( virtual/emacs )
+	expat? ( dev-libs/expat )
+	libffi? ( virtual/libffi )
+	gtk? ( >=x11-libs/gtk+-2.4:2 )
+	gtk3? ( x11-libs/gtk+:3 )
+	kde? ( >=kde-base/kdelibs-4 )
+	libedit? ( dev-libs/libedit )
+	libnotify? ( >=x11-libs/libnotify-0.4 )
+	m17n-lib? ( >=dev-libs/m17n-lib-1.3.1 )
+	ncurses? ( sys-libs/ncurses:0= )
+	nls? ( virtual/libintl )
+	qt4? ( dev-qt/qtgui:4[qt3support] )
+	skk? ( app-i18n/skk-jisyo )
+	sqlite? ( dev-db/sqlite:3 )
+	ssl? (
+		!libressl? ( dev-libs/openssl:0 )
+		libressl? ( dev-libs/libressl:0 )
+	)
```

Comment:
We should be using dev-libs/openssl:0= and dev-libs/libressl:0= in anticipation of future slot/subslot changes.  Both packages are sensitive to ABI changes as they begin to deprecate weak session/presentations like SSLv2 and SSLv3.